docs/pages/core_lib_doc/project_structure.md

```diff
@@ -7,39 +7,114 @@ folder: core_lib_doc
+    |       |        |          └─── <a href="#elastic">elastic</a> # store elastic search base classes
+    |       |        |          └─── <a href="#mongo">mongo</a> # store mongo base classes
+    |       |        └─── <a href="#data_access">data access</a> # expose API that uses data layer
+    |       |        |	    └─── <a href="#user">user</a> # grather user data access
+    |       |        |	    |    └─── <a href="#user_data_accesspy">user_data_access.py</a>
+    |       |        |	    |    └─── <a href="#user_list_data_accesspy">user_list_data_access.py</a>
+    |       |        |	    └─── some_data_access.py 
+    |       |        └─── <a href="#service">service</a> # expost the core-lib services
+    |       |               └─── <a href="#user-1">user</a> # gather user services
+    |       |               |    └─── <a href="#user_servicepy">user_service.py</a>
+    |       |        	    |    └─── <a href="#user_list_servicepy">user_list_service.py</a>
+    |       |        	    └─── <a href="#some_servicepy">some_service.py</a>
+    |       └─── <a href="#jobs">jobs</a> # jobs definitions
+    |       |	  └─── <a href="#update_something_jobpy">update_something_job.py</a>
+    |       |	  └─── <a href="#another_jobpy">another_job.py</a>
+    |       └─── <a href="#client">client</a> # client definitions
+    |       |	  └─── <a href="#server_a_clientpy">server_a_client.py</a>
+    |       |	  └─── <a href="#another_clientpy">another_client.py</a>
+    |       └─── <a href="#app_core_libpy">app_core_lib.py</a> # Main Core-Lib file that glows the entire library 
+    └─── <a href="#tests">tests</a> # Unit test the entire data_layers
+     	 └─── config # config file for the tests
+    </code>
+</pre>
 
+## Core-Lib Root Directory:
+This root directory serves as the foundational structure for the Core-Lib, encompassing all its components and resources. It acts as the primary container for organizing and managing the various modules, configurations, and assets that constitute the Core-Lib framework. Here, developers can find the core files and directories essential for building, configuring, and extending the functionality of the Core-Lib.
+
+### Core-Lib Main Directory:
+This subdirectory houses the main source code and modules of the Core-Lib, serving as the heart of the library. Developers will find key Python files and subdirectories containing the core functionalities and implementations that power the Core-Lib framework.
+
+### Tests:
+This directory contains the unit tests for the Core-Lib, allowing developers to verify the correctness and integrity of the library's functionalities. It includes test cases and configurations necessary for testing various components and features of the Core-Lib framework.
+
+## Config:
+This directory contains essential configuration files crucial for configuring and customizing the Core-Lib to suit specific requirements.
```

Comment:
This directory contains your Core-Lib configuration file crucial for configuring and customizing the Core-Lib to suit specific requirements.